### PR TITLE
Yahoo, Reddit etc now works correctly

### PR DESCRIPTION
**Pull request by vladest**
Test plan
- open Tablet->press Goto -> open Info card 
- enter an address in url input field
- make sure the page on address is opened

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: f88ba7bc5756d24775ec8960d891d015e40ff20a available at: https://gist.github.com/0f51690fea36cc11da79308192b018fc<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10160-f88ba7b.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10160-f88ba7b.dmg

File: interface/resources/qml/controls/TabletWebView.qml [hunk 1]
```diff
@@ -126,9 +126,20 @@ Item {
         view.setEnabled(true);
     }
 
+    function urlAppend(url) {
+        var _url = decodeURIComponent(url)
+        if (_url[_url.length - 1] !== "/")
+            _url = _url + "/"
+        if (currentPage === -1 || pagesModel.get(currentPage).webUrl !== _url) {
+            pagesModel.append({webUrl: _url})
+        }
```
Comment (druiz17) [inline interface/resources/qml/controls/TabletWebView.qml]: Only class local variables should have underscores

**Comment by hifi-gustavo** [comment]
Build console output for commit: a1892b351ff2680ed9786f4ed29b7550eb62e0fc available at: https://gist.github.com/0f51690fea36cc11da79308192b018fc<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10160-a1892b3.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10160-a1892b3.dmg

**Comment by 68chase** [comment]
Testing

**Comment by 68chase** [comment]
Passed 
hifi-snap-by-68chase-on-2017-04-08_16-45-22

**Pull request merged**